### ISSUE TITLE
Sorting out extra copies of Drummer left running

### ISSUE THREAD

**Issue by scripting** [scripting/drummerSupport#131]
There will be a pointer here shortly.

**Comment by akaKenSmith** [comment]
I'm willing to help test this when it's time.

**Comment by scotthansonde** [comment]
I can help test this. Sounds like a nice plumbing job!

**Comment by scripting** [comment]
@scotthansonde -- it _is_ plumbing.

**Comment by scripting** [comment]
@scotthansonde -- we're ready to begin the test. Is this the account you want to use for the test?

**Comment by scotthansonde** [comment]
@scripting Yes it is. I renamed my GitHub account a week ago. (Otherwise I'd be soon grandpapascott 😄 )

**Comment by scripting** [comment]
To be clear, it's the Twitter name that I need to know. My mistake. (Congrats on the addition to your family.)

**Comment by scripting** [comment]
Here's the first test of two.

1. You need two or more computers. 

2. Start Drummer in two tabs on the first machine.

3. Start Drummer on the second machine.

4. Verify that the first Drummer you opened on the first machine shows the About Drummer doc and the tab on the second machine shows Drummer. In other words you should see the original behavior.

Once you verify, I will let you know how to do the second test.

**Comment by scotthansonde** [comment]
Ah, the Twitter name is ScottHansonDE, that's unchanged. (And there's no planned addition to the family yet, but I'd be old enough for it!)

**Comment by scripting** [comment]
@scotthansonde -- please let me know when you've done the first test. I will then change the software so you'll get the new behavior.

**Comment by scotthansonde** [comment]
The first test was unsuccessful. I have two computers, kirby and kepler (Twins baseball players 😄 ). I first logged into kirby over Remote Desktop, started Drummer in Safari, and logged into ScottHansonDE. I then switched back to my MacBook Pro kepler. I opened Chrome, started Drummer. The previous instance did *not* show the About Drummer doc. (Screen shot shows Remote Desktop in the background, local Mac in the foreground.)

![CleanShot 2021-12-20 at 18 10 41](https://user-images.githubusercontent.com/8681117/146805855-f27039ff-d4ea-45a1-8a7a-c54984c11b3b.png)

**Comment by scripting** [comment]
I actually updated the app so you will not see the old behavior. So forget the first test.

The second test will follow...

**Comment by scripting** [comment]
If you're able to test the new functionality, post your Twitter username here, the one you use with Drummer.

**Comment by sabre23t** [comment]
Twitter user, sabre23t, successfully tested the new functionality.
First, I open drummer.scripting.com on my desktop. Then I open drummer.scripting.com on my phone. The copy on my desktop gets a dialog box _"Drummer is running in another tab. you must reload this one, or -- you can safely close it"_. I press OK on the dialog on the desktop, it reloads. Now, almost immediately the drummer copy on my phone gets similar _"Drummer is running in another tab..."_ dialog.
Works great. 😄

**Comment by akaKenSmith** [comment]
If it's okay to carry out the test with a computer and an iPad, I'm ready. Twitter: @KenSmith

**Comment by scripting** [comment]
@akaKenSmith -- yes of course -- the iPad is a computer.

**Comment by akaKenSmith** [comment]
Using a Macbook and an iPad, I was able to get the "you've got two tabs going" message successfully, as @sabre23t described above. Looks good from here, many thanks.
